assets/js/components/ClusterDetails/ClusterDetailsPage.jsx

```diff
@@ -34,33 +29,35 @@ export function ClusterDetailsPage() {
     return <div>Loading...</div>;
   }
 
-  const renderedNodes = cluster.details?.nodes?.map((node) => ({
-    ...node,
-    ...clusterHosts.find(({ hostname }) => hostname === node.name),
-  }));
-
   const hasSelectedChecks = cluster.selected_checks.length > 0;
 
-  return (
-    <ClusterDetails
-      clusterID={clusterID}
-      clusterName={getClusterName(cluster)}
-      selectedChecks={cluster.selected_checks}
-      hasSelectedChecks={hasSelectedChecks}
-      hosts={hosts}
-      clusterType={cluster.type}
-      cibLastWritten={cluster.cib_last_written}
-      sid={cluster.sid}
-      provider={cluster.provider}
-      clusterNodes={renderedNodes}
-      details={cluster.details}
-      lastExecution={lastExecution}
-      onStartExecution={(_, hostList, checks, navigateFunction) =>
-        dispatch(
-          executionRequested(clusterID, hostList, checks, navigateFunction)
-        )
-      }
-      navigate={navigate}
-    />
-  );
+  switch (cluster.type) {
+    case 'hana_scale_up':
+    case 'hana_scale_out':
+      return (
+        <HanaClusterDetails
+          clusterID={clusterID}
+          clusterName={getClusterName(cluster)}
+          selectedChecks={cluster.selected_checks}
+          hasSelectedChecks={hasSelectedChecks}
+          hosts={clusterHosts}
+          clusterType={cluster.type}
+          cibLastWritten={cluster.cib_last_written}
+          sid={cluster.sid}
+          provider={cluster.provider}
+          details={cluster.details}
+          lastExecution={lastExecution}
+          onStartExecution={(_, hostList, checks, navigateFunction) =>
+            dispatch(
+              executionRequested(clusterID, hostList, checks, navigateFunction)
+            )
+          }
+          navigate={navigate}
+        />
+      );
+    case 'ascs_ers':
+      return <div>ASCS/ERS</div>;
```

Comment:
Simple placeholder by now. I will have the `AscsErsClusterDetails` component next.
I just put some text that will be in the details view to have the tests working fine